content/index.md

```diff
@@ -3,5 +3,28 @@
 
 # Welcome to the Polympics Wiki
 
-Events:
+## Statement of Principles
+* Celebrate Polytopia
+* Encourage friendly rivalry between countries and geographic regions
+* Compete in a variety of Polytopia events (single player, multiplayer, multiplayer team, and art)
+* Award medals for the top three winners and a nitro prize for 1st place in each event
+
+## The Main Idea
+The summer Polympics is a multi-event Polytopia tournament inspired by the Olympics. Teams of up to 12 players will be formed based on nationality (or region in the case of very large nations like the US). Teams will send players to participate in a variety of event contests (single player, multiplayer, multiplayer team, and even art events). Medals will be assigned for each event and the team with the most medals will win the overall tournament.
+
+There are ten event contests, each having a set of commissioners (moderators).  All of the commissioners together are Polympic Committee. Commissioners will decide the rules for each event and moderate them.
+
+Players register through the Polympics website and join a team based on nationality. Participants will need a discord account. Teams will send players to compete in events. Some events allow multiple entries per team, others are limited to one representative (or pair of representatives) per team. See event rules for more details. Some events are mobile only while others allow for Steam or mobile users.
+
+## Events:
 - [Co-op Monument Rush](events/monument-rush)
+- [1v1 might] (events/1v1 might)
+- [2v2 team marathon] (events/2v2 team marathon)
+- [FFA relay]  (events/FFA relay)
+- [1v1 bullet] (events/1v1 bullet)
+- [1v1 glory/ Wicked Bender]  (events/1v1 glory- Wicked Bender)
+- [Art contest](events/Art contest)
+- [Skyscraper](events/Skyscraper)
+- [Perfect 10 ](events/Perfect 10)
+- [Apocalypse (1v3)](events/Apocalypse (1v3)
```

Comment:
The parts in the parenthesis are links. They should include only numbers, lowercase letters, dashes (`-`) and forward slashes (`/`) and should correspond to a file name. For example `events/monument-rush` corresponds to `content/events/monomuent-rush.md`. 

```suggestion
- [Co-op Monument Rush](events/monument-rush)
- [1v1 Might](events/1v1-might)
- [2v2 Team Marathon](events/2v2-team-marathon)
- [FFA Relay](events/ffa-relay)
- [1v1 Bullet](events/1v1-bullet)
- [1v1 Glory/Wicked Bender](events/1v1-glory-wicked-bender)
- [Art Contest](events/art-contest)
- [Skyscraper](events/skyscraper)
- [Perfect 10](events/perfect-10)
- [Apocalypse (1v3)](events/apocalypse-1v3)
```